cli/tests/integration_tests.rs

```diff
@@ -419,8 +420,38 @@ fn cache_test() {
   assert!(output.status.success());
   let out = std::str::from_utf8(&output.stdout).unwrap();
   assert_eq!(out, "");
-  // TODO(ry) Is there some way to check that the file was actually cached in
-  // DENO_DIR?
+
+  check_cached_file(module_url, std::path::PathBuf::from(deno_dir.path()))
+    .await;
+}
+
+async fn check_cached_file(module_url: url::Url, deno_dir: std::path::PathBuf) {
```

Comment:
This function is superfluous. Maybe just move this logic to the test that uses it?

Reply:
I was wondering if we can use the function to another test that requires checking the file. But since there's no other tests that require that, I'll merge to the test function. Thanks for suggestions.